core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -140,4 +140,10 @@ private TableProperties() {
 
   public static final String GC_ENABLED = "gc.enabled";
   public static final boolean GC_ENABLED_DEFAULT = true;
+
+  public static final String MAX_SNAPSHOT_AGE_MS = "history.expire.max-snapshot-age-ms";
+  public static final long MAX_SNAPSHOT_AGE_MS_DEFAULT = 5 * 24 * 60 * 60 * 1000; // 5 days
+
+  public static final String RETAIN_MIN_NUM_SNAPSHOTS = "history.expire.retain-min-num-snapshots";
+  public static final int RETAIN_MIN_NUM_SNAPSHOTS_DEFAULT = 1;
```

Comment:
This is not a particularly good default but we need it to keep the old behavior by default.